go.mod

```diff
@@ -131,3 +131,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/gravitl/netmaker => ../netmaker
```

Comment:
You don't want to merge this version of go.mod

Reply:
Fixed.